### DISCUSSION TITLE
looking one platform for SCRIPT+VOICE OVER+OVERLAYS  for my youtube channel

### DISCUSSION THREAD

**Discussion by kulbir1960** [community/community#66433] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

can i creat youtube scripte and all other stuffs on github?thanks

**Comment by Akash1134** [comment]
Welcome to the GitHub Community, @kulbir1960 , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category General and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Reply by keichigo** [reply]
Madharchod akash
